cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -4731,85 +4731,93 @@ ir.cpp:
-#  966|     r966_4(unsigned long)                 = Convert                         : r966_3
-#  966|     r966_5(unsigned long)                 = Constant[1]                     : 
-#  966|     r966_6(unsigned long)                 = Mul                             : r966_4, r966_5
-#  966|     r966_7(void *)                        = Call                            : func:r966_1, 0:r966_6
-#  966|     mu966_8(unknown)                      = ^CallSideEffect                 : ~mu959_4
-#  966|     r966_9(DefaultCtorWithDefaultParam *) = Convert                         : r966_7
-#  967|     r967_1(glval<unknown>)                = FunctionAddress[operator new[]] : 
-#  967|     r967_2(glval<int>)                    = VariableAddress[n]              : 
-#  967|     r967_3(int)                           = Load                            : &:r967_2, ~mu959_4
-#  967|     r967_4(unsigned long)                 = Convert                         : r967_3
-#  967|     r967_5(unsigned long)                 = Constant[4]                     : 
-#  967|     r967_6(unsigned long)                 = Mul                             : r967_4, r967_5
-#  967|     r967_7(void *)                        = Call                            : func:r967_1, 0:r967_6
-#  967|     mu967_8(unknown)                      = ^CallSideEffect                 : ~mu959_4
-#  967|     r967_9(int *)                         = Convert                         : r967_7
-#  968|     v968_1(void)                          = NoOp                            : 
-#  959|     v959_7(void)                          = ReturnVoid                      : 
-#  959|     v959_8(void)                          = UnmodeledUse                    : mu*
-#  959|     v959_9(void)                          = AliasedUse                      : ~mu959_4
-#  959|     v959_10(void)                         = ExitFunction                    : 
+#  959|     v959_1(void)                           = EnterFunction                   : 
+#  959|     mu959_2(unknown)                       = AliasedDefinition               : 
+#  959|     mu959_3(unknown)                       = InitializeNonLocal              : 
+#  959|     mu959_4(unknown)                       = UnmodeledDefinition             : 
+#  959|     r959_5(glval<int>)                     = VariableAddress[n]              : 
+#  959|     mu959_6(int)                           = InitializeParameter[n]          : &:r959_5
+#  960|     r960_1(glval<unknown>)                 = FunctionAddress[operator new[]] : 
+#  960|     r960_2(unsigned long)                  = Constant[40]                    : 
+#  960|     r960_3(void *)                         = Call                            : func:r960_1, 0:r960_2
+#  960|     mu960_4(unknown)                       = ^CallSideEffect                 : ~mu959_4
+#  960|     mu960_5(unknown)                       = ^InitializeDynamicAllocation    : &:r960_3
```

Comment:
I don't know why whitespace changed for this output. I recommend hiding whitespace changes when reviewing these changes.

Reply:
The whitespace between the result+type and the = is based on the longest result+type, which got longer by moving from `r966_9` to `r966_10`